Added Milestone 4 and 5 files

tshiu opened the pull request:
added 4 and 5 files

tshiu committed:
Added Milestone 4 and 5 files

mlezeng reviewed (changes requested):
I'm not seeing the program that asks the user for a number _n_ and writes the sume of the numbers 1 to _n_ in a file. Am I missing this?

I see some missing study drills, listed below:

Ex11:
- Write another 'form' like this to ask some other questions

Ex12:
- Any interesting notes about open, fil, os, and sys?

Ex13: 
- What happens when you try to give fewer than 3 arguments to your script? Can you explain the error?
- Write a script that has fewer arguments an done that has more. Make sure you give the unpacked variables good names
- Combine raw_input with argv to make a script that gets more input from a user

Ex14:
- Find out what Zork and Adventure were. Try to find a copy and play it
- Change the prompt variable to something else entirely

Ex15:
- Looks like there's an error in this code - line 13 print txt.read (). What error do you get when you run this script?
- What happens when you remove everything after print.txt.read() (line 13 in this script)
- What happens if you only use the script with raw_input instead of using argv? Why would one be better than the other?

Ex16: 
- Use strings, formats, and escapes to print out line 1, line 2, and line 3 with just one target.write() command instead of six
- If you open the file with 'w' mode, then do you really need target.truncate?

Ex17:
- What's the import function?
- Can you condense the in_file=open(from_file) and indata = infile.read() lines into one?
- What's the least lines of non-comment code you can condense this to? Right now, it's 15, the same as in the example.
- Why do you need to do output.close() at the end of the script?

Ex19:
- Add comments above each line, explaining in English what it does
- Write at least one more function of your own design, and run it 10 different ways

Ex20:
- the print_a_line function doesn't quite "print the specified line in a file" Can you figure out what it really does?
- What does the current_line variable in print_a_line do?
- Research the shorthand notation += and rewrite the script to use that

Ex21:
- What formula would recreate the set of operations in the puzzle at the end?

mlezeng commented:
For discussion during our MCM:

- How can what you learned during File I/O be applied at Clever?

tshiu committed:
update ex11.py

tshiu committed:
update ex11.py

tshiu committed:
update ex12.py

tshiu committed:
update ex13.py

tshiu committed:
update 14.py

tshiu committed:
Update ex18.py

tshiu committed:
Update ex16.py

tshiu committed:
Update ex17.py

tshiu committed:
Update ex14.py

tshiu committed:
Update PE_Exercise1.py

tshiu committed:
Update PE_Exercise2.py

tshiu committed:
Update ex16_sample.txt

tshiu committed:
Milestone 5 Deliverable

Write a program that asks the user for a number n and writes the sum of the numbers 1 to n into a file.

tshiu committed:
Milestone 3 Deliverable

Write a program that prints “‘Hello World’” to the screen.

tshiu committed:
Update ex17.py

tshiu committed:
Merge branch 'Milestone-2-and-3' into tshiu-patch-1
